api/v1alpha1/openapi.yaml

```diff
@@ -3031,6 +3031,56 @@ components:
               description: 'The directory in which the executable will be run from if it is left empty it will run from the users home directory.'
           required:
             - run
+    DeviceUpdatePolicySpec:
+      type: object
+      properties:
+        download:
+          $ref: '#/components/schemas/UpdatePolicy'
```

Comment:
DownloadPolicy? Assuming that this isn’t just schedule and would need to differ from update now/in the future.